### ISSUE TITLE
[SCREENREADER] Facility Locator has duplicate IDs

### ISSUE THREAD

**Issue by bkjohnson** [department-of-veterans-affairs/va.gov-team#13998]
## Description

This is a follow-up for #13647 

The facility-locator app has [some failures](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/13647-facility-locator/1/pipeline/) in the Cypress axe checker which need to be addressed. These seem to be limited to duplicate ids.

There is a branch already started for removing the exception: https://github.com/department-of-veterans-affairs/vets-website/compare/13647-facility-locator

## Point of Contact

@1Copenut 

## Acceptance Criteria

- [x] The temporary `_13647Exception` is removed from facility-locator Cypress tests
- [x] Facility Locator passes all Cypress axe checks

## Steps to reproduce

Here is one of the ways to end up with duplicate IDs

1. Go to `staging.va.gov/find-locations`
1. Type a search parameter into the location input (i.e., "Austin, TX")
1. Select "VA Health" for facility type
1. Click on the Search button
1. Notice that the `<h3>`s for the facilities in the results all have the id `facility-name`

## Possible Fixes (optional)

Incorporate the `index` prop into the rendered HTML instead of hardcoding the id to be `facility-result`.

https://github.com/department-of-veterans-affairs/vets-website/blob/dff41844c18e109e197b09479e2aa831ddb11771/src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx#L26-L34

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/2008881/94081696-05b65f00-fdb4-11ea-93a7-f68843ca1c3f.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by bkjohnson** [comment]
Closed by https://github.com/department-of-veterans-affairs/vets-website/pull/14393
